com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKLineVisual.cs

```diff
@@ -105,6 +104,15 @@ public float WidthMultiplier
         private XRRayInteractor rayInteractor;
 
         [SerializeField]
+        [Tooltip("The line renderer this visual has control over")]
+        private LineRenderer lineRenderer = null;
```

Comment:
Just to double check, this intentionally no longer has the `[HideInInspector]` attribute?

Reply:
Yeah it needs to be assigned since we're no longer putting the visual script on the same gameobject as the renderer